packages/api/src/models/organicHistoryModel.js

```diff
@@ -31,35 +30,17 @@ class OrganicHistory extends baseModel {
   static get jsonSchema() {
     return {
       type: 'object',
-      required: ['location_id', 'to_state', 'effective_date'],
+      required: ['location_id', 'organic_status', 'effective_date'],
       properties: {
-        id: { type: 'integer' },
+        organic_history_id: { type: 'string' },
         location_id: { type: 'string' },
-        to_state: { type: 'string' },
-        effective_date: { type: 'date-time' },
+        organic_status: { type: 'string', enum: ['Non-Organic', 'Transitional', 'Organic'] },
+        effective_date: { type: 'date' },
         ...this.baseProperties,
       },
       additionalProperties: false,
     };
   }
-
-  static get relationMappings() {
-    // Import models here to prevent require loops.
-    return {
-      location: {
-        relation: Model.BelongsToOneRelation,
```

Comment:
We only need relational mapping when we want to insert/update location using organicHistoryModel.upsertGraph. Then upsertGraph will be able to insert/update all tables that linked to locationModel. Should add relationMappings with caution. 
https://vincit.github.io/objection.js/api/query-builder/mutate-methods.html#upsertgraph